tasks/main.yml

```diff
@@ -14,6 +14,44 @@
     validate_certs: false
   when: not glpiinstalled.stat.exists or glpi_update
 
+- name: Perform basic installation (this take a while...)
```

Comment:
orthograph: this "takes a while", or "this may take a while"

Reply:
No soy bueno con el español.... Pondría la primer opción, porque lleva un tiempo...